### ISSUE TITLE
Yank packages?

### ISSUE THREAD

**Issue by baszalmstra** [mamba-org/quetz#419]
Is it possible to "yank" certain package versions? This means that when querying for package versions the yanked version is not listed but it is still available for download. This is more useful than removing a version because if a lock file somewhere references the yanked version it can still be installed.

**Comment by wolfv** [comment]
yes, that would be a really useful function. We don't have it currently, though.

PRs welcome!

**Comment by adriendelsalle** [comment]
is it similar to a `archived` status?

**Comment by wolfv** [comment]
@adriendelsalle the terminology comes from pypi: https://pypi.org/help/#yanked

@baszalmstra actually my answer was not entirely correct: if you use the repodata-patching plugin, you can add a special repodata patch package to your channel and the patch instructions can remove a package. You can look at how conda-forge does this here: https://github.com/conda-forge/conda-forge-repodata-patches-feedstock
